### DISCUSSION TITLE
If you are using this Action, do comment here

### DISCUSSION THREAD

**Discussion by smorimoto** [actions-ml/setup-ocaml#4] [Show and tell]
Just recording existing users of it to look at how it's being used.

**Comment by shonfeder** [comment]
I reported in another thread, but I've just started using this new version. So far so good. Many thanks for the hard work in improving the set up time!

**Comment by smorimoto** [comment]
All of the efforts and useful exploration results here are being pushed back to the upstream: https://github.com/avsm/setup-ocaml/pull/66
Please note that the upstream repository is going to be moved under the OCaml organization and this repository will be deprecated.
